app/services/export/s3_downloader.rb

```diff
@@ -0,0 +1,37 @@
+module Export
+  class S3DownloaderError < StandardError; end
+
+  class S3Downloader
+    def initialize(filename:)
+      @config = S3UploaderConfig.new(public: false)
+      @client = Aws::S3::Client.new(
+        region: config.region,
+        credentials: Aws::Credentials.new(
+          config.key_id,
+          config.secret_key
+        )
+      )
+      @filename = filename
+    end
+
+    attr_reader :client, :filename
+
+    def download
+      client.get_object(
+        bucket: config.bucket,
+        key: filename,
+        response_content_type: "text/csv"
+      ).body.read
+    rescue => error
+      raise_error(error.message)
+    end
+
+    private
+
+    attr_reader :config
+
+    def raise_error(original_message = nil)
+      raise S3DownloaderError, [original_message, "Error generating download for #{filename}"].join(" ")
```

Comment:
Guessing we care less about translations here since it's a bit of an admin thing?

Also, do we know that `original_message` has punctuation at the end? If not, `.join(" ")` might make for a messy message

Reply:
This may have slipped in from the previous spike - I think it probably should belong in a translation here.

Reply:
Have updated this to use a translation here (and the existing S3 upload file). We don't know about punctuation here (haven't been able to find the messages with a quick google), but we're following the same approach taken for the S3 Uploader class, so at least we're doing the same for both. As this is an error, I don't think we need to worry too much about it being a bit messy.